src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,34 @@
+import { useContext } from 'react';
+
+import { TodoContext } from '../../context/TodoContext';
+import { Status } from '../../types/Status';
+
+import { TodoItem } from '../TodoItem';
+
+export const TodoList: React.FC = () => {
+  const { todos, status } = useContext(TodoContext);
+
+  const updatedTodos = todos.filter(todo => {
```

Comment:
why do we need to iterate through the todos, when our status is `Status.All`?
better to add useMemo here, for some iterations/calculations

```
const updatedTodos = useMemo(() => {
  if (status === Status.All) return todos

  return todos.filter(...)
}, [todos])
```